admin/templates/configuration/tmp/configure/sql/qserv-worker.sql

```diff
@@ -1,6 +1,7 @@
--- Used by xrootd Qserv plugin:
+CREATE USER IF NOT EXISTS 'qsmaster'@'localhost';
```

Comment:
Yes this order matter, but only for mono-node configuration:

- qserv-czar.sql is loaded
- qserv-worker.sql is loaded and `qsmaster` account must not be removed in order to keep  access rights granted by previous sql script.